build.gradle

```diff
@@ -13,8 +13,8 @@ buildscript {
 	}
 
 	dependencies {
-		classpath 'io.spring.gradle:spring-build-conventions:0.0.28.RELEASE'
-		classpath "org.springframework.boot:spring-boot-gradle-plugin:$springBootVersion"
+		classpath 'io.spring.gradle:spring-build-conventions:0.0.34.RELEASE'
```

Comment:
There are a couple more steps involved when updating from 0.0.28.RELEASE.
We've done the upgrade in master via 5acb307a54c7a5347ffce8b4be7adb9bd3339aa5, which you can use a reference.

Reply:
I went ahead and backported the changes is 5acb307. If you rebase on master, the errors should be resolved.